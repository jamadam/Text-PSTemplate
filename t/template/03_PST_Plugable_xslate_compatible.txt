<ol>
    <% each($books, 'isbn')<<EOF2 %><li><% extract(&extract($hash, $isbn),'name') %> <% extract(&extract($hash, $isbn),'pages') %> pages / ISBN-13 : <% $isbn %></li>
    <% EOF2 %>
</ol>
