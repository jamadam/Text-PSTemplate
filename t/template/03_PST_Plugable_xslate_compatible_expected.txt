<ol>
    <li>プログラミングPerl Vol.1 708 pages / ISBN-13 : 978-4873110967</li>
    <li>プログラミングPerl Vol.2 1303 pages / ISBN-13 : 978-4873110974</li>
    <li>Программирование на Perl 1152 pages / ISBN-13 : 978-5932860205</li>
    <li>Programming Perl 1092 pages / ISBN-13 : 978-0596000271</li>
</ol>
